### ISSUE TITLE
Pubs that are rejoinders should not be deposited as peer reviews

### ISSUE THREAD

**Issue by gabestein** [pubpub/pubpub#2433]
### What went wrong, step-by-step?
1. Tried to deposit a Pub that is a rejoinder of several other pubs
2. It tried to deposit itself as a peer review, but Crossref rejected it because it didn't have an isReviewOf connection (only isReplyTo connections)

### What did you expect to happen?
Pub should be deposited as an article, not a review.

### What URL can we use to see the issue?
https://hdsr.mitpress.mit.edu/dash/pub/bd0tj6ub

### What browser/operating system were you using?

===

### Who reported this? (If not you)
Amara / HDSR

**Comment by pilotSaad** [comment]
Correct. [Rejoinders](<a href="https://northlandz.com/outdoor-trains">outdoor train<a) , which are responses to specific points raised in a publication, are not considered peer reviews. Peer reviews are evaluations of a manuscript conducted by independent experts in the same field prior to its publication, with the aim of improving the quality and validity of the work. Rejoinders, on the other hand, are a form of post-publication discussion and do not play the same role as peer reviews in the scientific publishing process.

**Comment by pilotSaad** [comment]
Correct. Rejoinders, which are responses to specific points raised in a publication, are not considered peer reviews. Peer reviews are evaluations of a manuscript conducted by independent experts in the same field prior to its publication, with the aim of improving the quality and validity of the work. Rejoinders, on the other hand, are a form of post-publication discussion and do not play the same role as peer reviews in the scientific publishing process.

[Bug](<a href="https://northlandz.com/outdoor-trains">outdoor train<a)
